apps/public-data/src/public-data/dto/get-one-post-category.dto.ts

```diff
@@ -0,0 +1,8 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNumber } from 'class-validator';
+
+export class GetOnePostCategoryDataDto {
```

Comment:
이름에서 -data를 사용하는 습관은 제거해보시면 좋을 것 같습니다.
클린코드에서는 요런 접미사를 지양하라고 얘기하기도 해요. [참고](https://ostarblog.tistory.com/6)
```suggestion
export class GetOnePostCategoryDto {
```

Reply:
넵! 코드를 계속 보다보니까 확실히 의미가 없는 접미사나 단어를 많이 채택하는
것 같습니다.... 지양하면서 더 좋은 네이밍을 하도록 노력하겠습니다!